CHANGELOG

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Wed Nov 30 15:01:51 UTC 2016 - joseivanlopez@gmail.com
+
+- Improve deletion of elements in the augeas parser
```

Comment:
@joseivanlopez In your case that ID would be "bsc#983486".

@jreidinger, @joseivanlopez is working in fixing ntp-client just because I told him so. I never gave the bug id to him. And it's private anyways :-)